scss/utilities/icon-mixin.scss

```diff
@@ -1,4 +1,4 @@
-@mixin pixelize($matrix, $colors, $size, $default-color: null) {
+@mixin pixelize($matrix, $colors, $default-color: null) {
```

Comment:
I think being able to reduce the size of the entire framework is amazing :tada: ! To me it's really clever what you've done here, I never thought about it, just awesome work. 

I've created most of the pixel arts just for the fun of it and maybe to attract more users :smile: .
In my environment, I've forked your repo to test it and I notice that you changed the `icon-mixin.scss`, but checkboxes, lists and radio buttons use that aswell and they got bugged here, not sure if I'm missing something.

Radio:

![image](https://user-images.githubusercontent.com/22016005/50256040-aad02680-03db-11e9-932f-f4f1145486b4.png)

Checkbox:

![image](https://user-images.githubusercontent.com/22016005/50256053-b3c0f800-03db-11e9-8e41-ca6dc1a0d354.png)

Lists:

![image](https://user-images.githubusercontent.com/22016005/50256064-bb809c80-03db-11e9-97e1-0b9939abcbdd.png)

I've tested all the rest and LGTM.

Reply:
Should be fixed now, I forgot to set their `font-size` and remove the size from the `pixelize` call